traits/observation/tests/test_expression.py

```diff
@@ -183,6 +184,52 @@ def test_join_expressions(self):
         self.assertEqual(actual, expected)
 
 
+class TestObserverExpressionAnytrait(unittest.TestCase):
+    """ Test anytrait function and method. """
+
+    def test_anytrait_function_notify_true(self):
+        expr = expression.anytrait(notify=True)
+        expected = [
+            create_graph(
+                FilteredTraitObserver(filter=anytrait_filter, notify=True),
+            ),
+        ]
+        actual = expr._as_graphs()
+        self.assertEqual(actual, expected)
+
+    def test_anytrait_function_notify_false(self):
+        expr = expression.anytrait(notify=False)
+        expected = [
+            create_graph(
+                FilteredTraitObserver(filter=anytrait_filter, notify=False),
+            ),
+        ]
+        actual = expr._as_graphs()
+        self.assertEqual(actual, expected)
+
+    def test_anytrait_method_notify_true(self):
+        expr = expression.trait("name").anytrait(notify=True)
+        expected = [
+            create_graph(
+                NamedTraitObserver(name="name", notify=True, optional=False),
+                FilteredTraitObserver(filter=anytrait_filter, notify=True),
+            ),
+        ]
+        actual = expr._as_graphs()
+        self.assertEqual(actual, expected)
+
+    def test_anytrait_method_notify_false(self):
```

Comment:
there seems to be a copy-paste error here - the test method is different but the bodies look the same.

Reply:
Thank you! Fixed, I think.